conda_build/render.py

```diff
@@ -266,14 +266,15 @@ def get_upstream_pins(m, actions, env):
                 with open(downstream_file + '.yaml') as f:
                     specs = yaml.safe_load(f)
         if not specs and os.path.isfile(pkg_loc):
-            legacy_specs = utils.package_has_file(pkg_loc, 'info/run_exports')
```

Comment:
This order swap was done in the hopes of slightly speeding up this process.  Packages with run_exports.yaml will be in the majority, so hits on those should be a bit faster.